ui/cypress/e2e/collectors.test.ts

```diff
@@ -165,5 +177,64 @@ describe('Collectors', () => {
       cy.getByTestID('resource-card').should('contain', secondTelegraf)
       cy.getByTestID('resource-card').should('not.contain', thirdTelegraf)
     })
+
+    describe('sorting', () => {
+      const telegrafs = ['b', 'a', 'c']
+      const bucketz = ['MO_buckets', 'EZ_buckets', 'Bucky']
+      beforeEach(() => {
+        const [firstTelegraf, secondTelegraf, thirdTelegraf] = telegrafs
+        const description = 'Config Description'
+        const [firstBucket, secondBucket, thirdBucket] = bucketz
+
+        cy.get('@org').then(({id}: Organization) => {
+          cy.createTelegraf(firstTelegraf, description, id, firstBucket)
+          cy.createTelegraf(secondTelegraf, description, id, secondBucket)
+          cy.createTelegraf(thirdTelegraf, description, id, thirdBucket)
+        })
+
+        cy.reload()
+      })
+      // sort by buckets test here
+      it('can sort telegraf configs by bucket', () => {
+        cy.getByTestID('bucket-name').should('have.length', 3)
+        cy.getByTestID('bucket-sorter').click()
+
+        bucketz.sort()
+        cy.getByTestID('bucket-name')
+          .each((val, index) => {
+            const text = val.text()
+            expect(text).to.equal(`Bucket: ${bucketz[index]}`)
```

Comment:
Might want to do a less strict match here.  `Bucket: ` is a presentational concern and is highly likely to change.  Something like `includes` instead of `equal`